examples/use_tools.py

```diff
@@ -8,21 +8,38 @@
 import unifai
 from typing import List
 
-tools = unifai.Tools(api_key=os.getenv("UNIFAI_AGENT_API_KEY", ""))
-
-async def run(msg: str):
+async def run(msg: str, toolkit_id=None):
+    agent_api_key = os.getenv("UNIFAI_AGENT_API_KEY", "")
+    tools = unifai.Tools(api_key=agent_api_key)
+  
+    if toolkit_id:
```

Comment:
available_tools will not be empty since we set dynamic_tools=True.
L15-27 can be simplified to
```
available_tools = await tools.get_tools_async(
    dynamic_tools=True,
    static_toolkits=[toolkit_id] if toolkit_id else [],
)
```